x/gov/types/vote.go

```diff
@@ -53,6 +55,33 @@ func (v Vote) Empty() bool {
 	return v.String() == Vote{}.String()
 }
 
+// NewNonSplitVoteOption creates a single option vote with weight 1
+//nolint:interfacer
+func NewNonSplitVoteOption(option VoteOption) WeightedVoteOptions {
+	return WeightedVoteOptions{{option, sdk.NewDec(1)}}
+}
+
+func (v WeightedVoteOption) String() string {
+	out, _ := json.Marshal(v)
```

Comment:
it the output different here between `json.Marshal` and `proto.Marshal`?

Reply:
Used proto.Marshal

Reply:
Why do we want to use proto.Marshal?  Shouldn't string be JSON, so it's human readable?

Reply:
oops, my bad, I was thinking `json.Marshal` vs `ProtoMarshalJSON`

Reply:
reverted back to json.Marshal